examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -1908,12 +1979,6 @@ def get_sigmas(timesteps, n_dim=4, dtype=torch.float32):
 
                 # Sample noise that we'll add to the latents
                 noise = torch.randn_like(model_input)
-                if args.noise_offset:
-                    # https://www.crosslabs.org//blog/diffusion-with-offset-noise
```

Comment:
Why are we removing this?

Reply:
I just realize it doesn't really mean much for _sdxl_ Loras, but could be discussed in a separate PR as these are unrelated

Reply:
this cannot be removed @linoytsaban